lib/executor/constant_vus.go

```diff
@@ -176,15 +176,26 @@ func (clv ConstantVUs) Run(parentCtx context.Context, out chan<- stats.SampleCon
 	regDurationDone := regDurationCtx.Done()
 	runIteration := getIterationRunner(clv.executionState, clv.logger)
 
+	maxDurationCtx = lib.WithScenarioState(maxDurationCtx, &lib.ScenarioState{
+		Name:       clv.config.Name,
+		Executor:   clv.config.Type,
+		StartTime:  startTime,
+		ProgressFn: progressFn,
+	})
+
 	returnVU := func(u lib.InitializedVU) {
 		clv.executionState.ReturnVU(u, true)
 		activeVUs.Done()
 	}
+
+	// Channel for synchronizing scenario-specific iteration increments
```

Comment:
This comment seems to be for an earlier version of the PR?

Reply:
:facepalm: Merge gone awry... It was also in `per_vu_iterations.go`. Fixed in latest push.

Some of the intermediate commits are definitely broken, but :shrug: